server/api/classyear/classyear.controller.js

```diff
@@ -12,7 +12,7 @@ exports.index = function(req, res) {
   	"current": true
   }, function (err, classYear) {
   	if(err) { return handleError(res, err); }
-   	return res.json(200, classYear);
+   	return res.status(200).json(classYear);
```

Comment:
Can these backend fixes be split into a different PR? They are unrelated (and causing conflicts)